community/hangouts/index.md

```diff
@@ -33,19 +33,17 @@ The Monero community is diverse and varied. We come from all over, but we defini
                         <div class="info-block">
                             <div class="row center-xs">
```

Comment:
Not sure if this should be moved below the IRC channels section because it's a lot shorter, and references the text below.